### ISSUE TITLE
[Bug][a11y] Few options of menu not accessible while using app through Talkback only

### ISSUE THREAD

**Issue by VaishnaviShri** [mozilla-mobile/fenix#18968]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce

- Have Talkback switched on
- Launch the app and select the three dots for Menu
- Try to navigate to the "Settings" or "Save to collection" option guided by Talkback only. (Closing your eyes and navigating might help understand the issue)

### Expected behavior
While using the app through Talkback only there should be information conveyed about a few more options being present in the menu other than ones on the screen. This would give an idea of the menu list being scrollable which might not be intuitive for a lot of users, especially ones with some sort of visual disability.

### Actual behavior
There is no way to access the "Settings" and few other options in the menu unless you can see the half displayed option "Add to home screen" or are aware of the remaining options of the menu being accessible after scrolling.
This behavior not being very intuitive has been mentioned in #18914 but I feel the issue should be looked at from the perspective of enhancing the accessibility of the app too.

Video of the above-mentioned steps (please turn up the volume):

https://user-images.githubusercontent.com/58984760/114451152-8d4ea880-9bf4-11eb-854c-94d93fa9cb34.mp4

### Device information

* Device vendor / model and Android version: Redmi Note 7 Pro and Android 10
* Firefox for Android version: 1.0.2114 (Build # 1)
AC: 75.0.20210402143051, 0e4d25a96
GV: 89.0a1-20210402094809
AS: 74.0.1

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-10352)

**Comment by VaishnaviShri** [comment]
I would like to work on this issue.
